backend/src/test/java/ch/puzzle/okr/service/TeamServiceTest.java

```diff
@@ -74,4 +75,58 @@ void shouldNotFindTheTeam() {
         assertEquals(404, exception.getRawStatusCode());
         assertEquals("Team with id 6 not found", exception.getReason());
     }
+
+    @Test
+    void shouldSetNameWhenCreatingTeam() {
+        Team team = Team.Builder.builder().withName("TestTeam").build();
+        Mockito.when(teamRepository.save(any())).thenReturn(team);
+
+        Team savedTeam = teamService.saveTeam(team);
+        assertNull(savedTeam.getId());
+        assertEquals(savedTeam.getName(), "TestTeam");
+    }
+
+    @Test
+    void shouldThrowResponseStatusExceptionWhenPuttingId() {
+        Team team = Team.Builder.builder().withId(2L).withName("TestTeam").build();
+
+        ResponseStatusException exception = assertThrows(ResponseStatusException.class, () -> {
+            teamService.saveTeam(team);
+        });
+        assertEquals(exception.getStatus(), HttpStatus.BAD_REQUEST);
+        assertEquals(exception.getMessage(), "400 BAD_REQUEST \"Not allowed to give an id\"");
```

Comment:
Bitte in der ganzen Klass anpassen:
1. bei den assertEquals wird zuerste der `Erwartete Wert` und anschliessend der `AktuellZurückgelieferte Wert` angegeben.
2. statt  `exception.getMessage()` wäre schöner `assertEquals("Not allowed to give an id", exception.getReason());`